lampetia-security-model/src/main/scala/lampetia/security/model/models.scala

```diff
@@ -279,7 +298,7 @@ trait SecurityModel {
       }
       object parentResource extends Composite[Option[Resource]] {
         val resourceId =
-          property[Option[ResourceId]]("resourceId")
```

Comment:
I can't see this, permission still has bit(32) type:

models.scala line 310:

val permission =
property[Permission]("permission").set(sql.`type`("bit(32)"))

but I have changed the consumer to :
implicit lazy val consumePermission: Consume[Permission] = consume[String
].fmap(r => Permission(Integer.parseUnsignedInt(r, 2)))

to convert from binary to decimal


On Fri, Aug 7, 2015 at 6:44 PM, Hossam Karim <notifications@github.com>
wrote:

> In
> lampetia-security-model/src/main/scala/lampetia/security/model/models.scala
> <https://github.com/hkarim/lampetia/pull/2#discussion_r36536926>:
>
> > @@ -279,7 +298,7 @@ trait SecurityModel {
> >        }
> >        object parentResource extends Composite[Option[Resource]] {
> >          val resourceId =
> > -          property[Option[ResourceId]]("resourceId")
>
> Did you change the permission type from bit(32) to something else? Why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hkarim/lampetia/pull/2/files#r36536926>.
>



-- 
Best Regards,
Radwa Osama